include/grpcpp/ext/otel_plugin.h

```diff
@@ -100,6 +100,13 @@ class OpenTelemetryPluginBuilder {
   OpenTelemetryPluginBuilder& SetGenericMethodAttributeFilter(
       absl::AnyInvocable<bool(absl::string_view /*generic_method*/) const>
           generic_method_attribute_filter);
+  // Methods to manipulate which instruments are enabled in the OpenTelemetry
+  // Stats Plugin.
+  OpenTelemetryPluginBuilder& EnableMetrics(
+      const std::vector<absl::string_view>& metric_names);
```

Comment:
`absl::Span<>` might be another choice? It would allow the API to take more types of input (e.g. C-style array, initializer list) without constructing a vector.

```
OpenTelemetryPluginBuilder& EnableMetrics(
      absl::Span<const absl::string_view> metric_names);
```

Reply:
Suggestion taken, thanks!